certified-connectors/RAPID Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,645 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "1.15.0",
+		"title": "RAPID Platform (V1)",
```

Comment:
@MattFranklin-Rapid The info section has `version` property. Can you please let us know why you want to add `"(V1)"` in the title to explicitly show the connector version.

Reply:
Updated to drop (V1). During the engineering meeting our head architect was wanting to differentiate between the current v1 of our RAPID Api and the in development v2. The current intention is to differentiate that within the actions rather than as a potential additional future connector.